tests_integration_with_local_chain/nft_agents/test_agent_communication.py

```
@@ -33,20 +34,22 @@
             agent_address=mock_agent_address, web3=local_web3
         )
-        == 1
+        == initial_messages + 1
     )
 
 
 def test_pop_message(local_web3: Web3) -> None:
+    #### Delete me after test passes
     keys = APIKeys()
-    mock_agent_address = keys.bet_from_address
+    mock_agent_address = Web3.to_checksum_address(accounts[2].address)
```

Ruff (0.8.2)

Undefined name `accounts`

(F821)